pytest_rpc/__init__.py

```diff
@@ -65,8 +111,14 @@ def pytest_runtestloop(session):
             junit_xml_config = getattr(session.config, '_xml', None)
 
             if junit_xml_config:
-                for env_var in ENV_VARS:
-                    junit_xml_config.add_global_property(env_var, os.getenv(env_var, 'Unknown'))
+                runner = _get_ci_environment(session)
```

Comment:
The `runner` variable should be changed to `ci_environment` to avoid future confusion.

Reply:
Fixed